model/src/main/java/oracle/kubernetes/weblogic/domain/ClusterConfigurator.java

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Copyright 2018,2019, Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
Mark has been teaching me to be more pedantic about these copyrights :(...  Apparently, there needs to be a space as in, "Copyright 2018, 2019,"